src/rdkafka.h

```diff
@@ -7085,6 +7127,46 @@ typedef enum rd_kafka_AclOperation_t {
 /**@}*/
 
 
+/**
+ * @name ListOffsets API
+ * @brief Given a topic_partition list, provides the offset information.
+ * @{
+*/
+
+/**
+ * OffsetSpec for Earliest, Latest and Timestamp of the Offset to list via ListOffsets.
+*/
+typedef enum rd_kafka_OffsetSpec_t{
+    RD_KAFKA_OFFSET_SPEC_MAX_TIMESTAMP = -3,
+    RD_KAFKA_OFFSET_SPEC_EARLIEST = -2,
+    RD_KAFKA_OFFSET_SPEC_LATEST = -1,
+}rd_kafka_OffsetSpec_t;
+
+/**
+ * Defines a ListOffsetResultInfo to be created.
+*/
+typedef struct rd_kafka_ListOffsetResultInfo_s rd_kafka_ListOffsetResultInfo_t;
+
+/**
+ * Returns the rd_kafka_topic_partition_t of the rd_kafka_ListOffsetResultInfo_t passed.
```

Comment:
```suggestion
 * @brief Returns the topic partition of the passed \p result_info.
```
add `@brief` in other comments too.